doc/configcommands.dsv

```diff
@@ -40,6 +40,14 @@ feedhq-url||<url>||"https://feedhq.org/"||Configures the URL where your FeedHQ i
 feedlist-format||<format>||"%4i %n %11u %t"||This variable defines the format of entries in the feed list. See the respective section in the documentation for more information on format strings.||feedlist-format " %n %4i - %11u -%> %t"
 feedlist-title-format||<format>||"%N %V - %?F?Feeds&Your feeds? (%u unread, %t total)%?F? matching filter `%F'&?%?T? - tag `%T'&?" (localized)||Format of the title in feed list. See "Format Strings" section of Newsboat manual for details on available formats.||feedlist-title-format "Feeds (%u unread, %t total)"
 filebrowser-title-format||<format>||"%N %V - %?O?Open File&Save File? - %f" (localized)||Format of the title in file browser. See "Format Strings" section of Newsboat manual for details on available formats.||filebrowser-title-format "%?O?Open File&Save File? - %f"
+freshrss-flag-star||<flag>||""||If set and FreshRSS support is used, then all articles that are flagged with the specified flag are being "starred" in FreshRSS and appear in the list of "Starred items".||freshrss-flag-star "b"
+freshrss-login||<login>||""||This variable sets your FreshRSS login for FreshRSS support.||freshrss-login "your-login"
+freshrss-min-items||<number>||20||This variable sets the number of articles that are loaded from FreshRSS per feed.||freshrss-min-items 100
+freshrss-password||<password>||""||This variable sets your FreshRSS password for FreshRSS support. Double quotes and backslashes within it <<#_using_double_quotes,should be escaped>>.||freshrss-password "here_goesAquote:\""
+freshrss-passwordfile||<path>||""||A more secure alternative to the above, by storing your password elsewhere in your system.||freshrss-passwordfile "~/.newsboat/freshrss-pw.txt"
+freshrss-passwordeval||<command>||""||Another secure alternative, is providing your password from an external command that is evaluated during login. This can be used to read your password from a gpg encrypted file or your system keyring.||freshrss-passwordeval "gpg --decrypt ~/.newsboat/freshrss-password.gpg"
+freshrss-show-special-feeds||[yes/no]||yes||If set and FreshRSS support is used, then a "Starred items" feed (containing your starred/favourited articles) appears in your subscription list.||freshrss-show-special-feeds "no"
+freshrss-url||<url>||"https://freshrss.org/"||Configures the URL where your FreshRSS instance resides.||freshrss-url "https://freshrss.example.com/"
```

Comment:
Does FreshRSS really provide a hosted instance on freshrss.org? I see they have a demo on a separate domain, but I don't see any indication that they have a hosted version. If they don't, Newsboat:
1. shouldn't provide a default for `freshrss-url`;
2. should perform checks like https://github.com/newsboat/newsboat/blob/f7d8ab4ac8d85b6947b150ff8b3388c89dd9f730/src/controller.cpp#L286-L305 to ensure that all the necessary settings are configured.

(There's some repetition in that code already, it's probably time to generalize it a bit… But that's for another PR.)